Sources/NIOSSL/ByteBufferBIO.swift

```diff
@@ -12,9 +12,14 @@
 //
 //===----------------------------------------------------------------------===//
 
-@_implementationOnly import CNIOBoringSSL
 import NIOCore
 
+#if compiler(>=6.1)
```

Comment:
We could do internal imports from 6.0, so why 6.1, did something change?

Reply:
Yeah, we started getting warnings, which was going to inhibit warnings as errors.